### PR TITLE
Multi arch fbc 4.14

### PR DESCRIPTION
**Pull request by marioferh**

### CODE REVIEW

File: .tekton/coo-fbc-v4-14-pull-request.yaml [hunk 1]
```diff
@@ -224,7 +247,7 @@ spec:
         value: $(params.build-image-index)
       - name: IMAGES
         value:
-        - $(tasks.build-container.results.IMAGE_URL)@$(tasks.build-container.results.IMAGE_DIGEST)
+        - $(tasks.build-images.results.IMAGE_REF[*])
       runAfter:
       - build-container
```
Comment (danielmellado) [inline .tekton/coo-fbc-v4-14-pull-request.yaml:252]: this should be build-images. note that build-container no longer exists.

Comment (marioferh) [reply .tekton/coo-fbc-v4-14-pull-request.yaml]: done

File: .tekton/coo-fbc-v4-14-pull-request.yaml [hunk 2]
```diff
@@ -176,10 +195,13 @@ spec:
         - "true"
       workspaces:
       - name: output
```
Comment (danielmellado) [inline .tekton/coo-fbc-v4-14-pull-request.yaml:197]: note the wrong change here, please do keep basic-auth, you should delete just the output workspace

**Review by danielmellado — COMMENTED** [review]
Please do take a look at the whole pipelineRun before they merging, it's easy to lost track of small nits here. Thanks!

File: .tekton/coo-fbc-v4-14-pull-request.yaml (hunk 2 above)
Comment (marioferh) [reply .tekton/coo-fbc-v4-14-pull-request.yaml:197]: done

File: .tekton/coo-fbc-v4-14-push.yaml [hunk 3]
```diff
@@ -172,11 +191,15 @@ spec:
         values:
         - "true"
       workspaces:
-      - name: output
-        workspace: workspace
       - name: basic-auth
         workspace: git-auth
     - name: build-container
```
Comment (danielmellado) [inline .tekton/coo-fbc-v4-14-push.yaml:196]: this is still wrong, build-container should be renamed to build-images. I assume you meant to *remove* this line, as I see build-images later in the code.

**Review by danielmellado — CHANGES_REQUESTED** [review]
Left a comment, please do review the code changes thoroughly as they're hard to notice later on ;)

File: .tekton/coo-fbc-v4-14-push.yaml (hunk 3 above)
Comment (marioferh) [reply .tekton/coo-fbc-v4-14-push.yaml]: done

**Pull request merged**